internal/backend/sftp/sftp.go

```diff
@@ -287,6 +287,20 @@ func tempSuffix() string {
 	return hex.EncodeToString(nonce[:])
 }
 
+func setReadOnly(client *sftp.Client, path string, mode os.FileMode) error {
```

Comment:
A minor thing, but could we call it setFileReadOnly to match the function name used in backend/local, so that filename search finds the two corresponding functions together (as with the other function names here)?